_data/LuisEduardoRamos/LuisEduardoRamos.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Luis Eduardo Ramos
+institution: Instituto Tecnologico de Culiacan.
+profile_pic: luisramos.JPG
```

Comment:
Git is case-sensitive. Could you please fix the file name? You could do one of the following:
1. Change `luisramos.JPG` to `luisramos.jpg ` in the markdown file.
2. Rename `luisramos.jpg ` to `luisramos.JPG`.
I'd suggest you change the name in the markdown file!